sdk/storage/Azure.Storage.Blobs/src/Sas/ContainerSasPermissions.cs

```diff
@@ -93,7 +70,6 @@ public override string ToString()
         /// <returns>The parsed <see cref="ContainerSasPermissions"/>.</returns>
         public static ContainerSasPermissions Parse(string s)
         {
-            // Clear the flags
             var p = new ContainerSasPermissions(); 
```

Comment:
all these `p`, `s`, `c` usage is crazy. Could we add proper names to the variables and make the code readable?

Reply:
The `string s` is actually standard for .NET.  I wanted to change those to `text` or `value` since they're publicly exposed, but all the core `Parse` methods call their parameter `s`.

As for the local variables, it's the same as above - I'm not trying to make global style changes as part of the API work.